src/components/LayoutStepper.vue

```diff
@@ -11,7 +11,7 @@
                 overlay
                 side="right"
                 v-model="isPreviewDrawerEnabled"
-                width="600"
+                v-bind:width="width"
```

Comment:
```suggestion
                v-bind:width="600"
```
I think you can just use 600 instead because of the `v-bind`. It should be parsed as a number.